### ISSUE TITLE
Update dependency ytdl-core to more recent version

### ISSUE THREAD

**Issue by NyWn** [skiptirengu/media-split#9]
Running `media-split` from a fresh install errors with:
```
Error: Cannot find module 'm3u8stream/lib/parse-time'
Require stack:
- <path>/ytdl-core/lib/info-extras.js
- <path>/node_modules/ytdl-core/lib/info.js
- <path>/node_modules/ytdl-core/lib/index.js
- <path>/node_modules/media-split/lib/MediaSplit.js
```
`media-split` currently relies on `ytdl-core` version 0.29.7, which imports `parseTime` from `m3u8stream/lib/parse-time`, however `m3u8stream` has since renamed `/lib/` to `/dist/`.

Updating the `ytdl-core` dependency to the version containing the change (`^1.0.0`) will fix.

See: 
- https://github.com/fent/node-ytdl-core/issues/519

Commit in `ytdl-core` updating path:
- https://github.com/fent/node-ytdl-core/commit/0d730d267801c6048864eedc05ee0bf2d5e43d92

**Comment by NyWn** [comment]
Noticed package.json in repo has later version of `2.0.1` but my local has a much earlier version, clearly not an issue with the repo. Apologies.

**Comment by NyWn** [comment]
Reopening: The media-split package in the npm registry needs updating as it depends on the old version of ytdl-core. Looks like its not been updated in about a year which is why I was seeing the errors.

**Comment by sugiruu** [comment]
Fixed on media-split@5.1.1
